src/main.h

```diff
@@ -119,6 +119,8 @@ static const bool DEFAULT_TXINDEX = true;
 static const bool DEFAULT_ADDRESSINDEX = false;
 static const bool DEFAULT_TIMESTAMPINDEX = false;
 static const bool DEFAULT_SPENTINDEX = false;
+static const unsigned int DEFAULT_DB_MAX_OPEN_FILES = 1000;
```

Comment:
Bitcoin Core had some problems with high number of open files in the past and they still have it at 64 so this change can be dangerous actually.

Let's make sure it works on every platform in all modes (sending a bunch of txes, reindexing, syncing from network) first or change this back to current values i.e.
```
static const unsigned int DEFAULT_DB_MAX_OPEN_FILES = 64;
static const bool DEFAULT_DB_COMPRESSION = false;
```